bridges/snowbridge/parachain/runtime/polkadot-common/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+//! # Polkadot Common
+//!
+//! Config used for the Polkadot asset hub and bridge hub runtimes.
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::parameter_types;
+use xcm::opaque::lts::NetworkId;
+
+pub const INBOUND_QUEUE_MESSAGES_PALLET_INDEX: u8 = 80;
+
+parameter_types! {
+	// Network and location for the Ethereum chain.
+	pub EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 1 };
```

Comment:
Not sure if we should actually define this here and not in the `runtimes` repo?

Reply:
We have deliberated about this quite a bit and would prefer to contain all the Snowbridge runtime config in our crates, so it is easy to see the config and update them all in a similar place.

Reply:
agree with @ggwpez,
I also think that this kind of constant should be directly in the runtime,
e.g. when you add this to the `polkadot-fellows/runtimes` repo, fellows should be able to set/change/control `chain_id` - which Ethereum to connect to/

Reply:
Ok, fair enough. Can I add it in the [bridge_to_ethereum](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_ethereum_config.rs).rs module?

Reply:
@claravanstaden exactly :)

Reply:
@bkontur so I remember now we put the config in a central place. We use `INBOUND_QUEUE_MESSAGES_PALLET_INDEX` in the asset hub runtime as well as in the bridge hub runtime. So I can't add it to [bridge_to_ethereum.rs](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_ethereum_config.rs) because then it won't be available to use in asset hub.

How about I add a snowbridge.rs module in `polkadot-sdk/cumulus/parachains/common/src`?

Reply:
Or even in `polkadot-sdk/cumulus/parachains/common/src/rococo.rs`?

Reply:
I ended up doing this ☝🏻